v1/python-sdk/tutorials/automl-with-azureml/forecasting-hierarchical-timeseries/auto-ml-forecasting-hierarchical-timeseries.ipynb

```diff
@@ -300,24 +314,26 @@
     "label_column_name = \"quantity\"\n",
     "forecast_horizon = 7\n",
     "\n",
+    "forecasting_parameters = ForecastingParameters(\n",
+    "    time_column_name=time_column_name,\n",
+    "    forecast_horizon=forecast_horizon,\n",
+    ")\n",
     "\n",
-    "automl_settings = {\n",
-    "    \"task\": \"forecasting\",\n",
-    "    \"primary_metric\": \"normalized_root_mean_squared_error\",\n",
-    "    \"label_column_name\": label_column_name,\n",
-    "    \"time_column_name\": time_column_name,\n",
-    "    \"forecast_horizon\": forecast_horizon,\n",
-    "    \"hierarchy_column_names\": hierarchy,\n",
-    "    \"hierarchy_training_level\": training_level,\n",
-    "    \"track_child_runs\": False,\n",
-    "    \"pipeline_fetch_max_batch_size\": 15,\n",
-    "    \"model_explainability\": model_explainability,\n",
-    "    \"n_cross_validations\": \"auto\",  # Feel free to set to a small integer (>=2) if runtime is an issue.\n",
-    "    \"cv_step_size\": \"auto\",\n",
+    "automl_settings = AutoMLConfig(\n",
+    "    task=\"forecasting\",\n",
+    "    primary_metric=\"normalized_root_mean_squared_error\",\n",
+    "    experiment_timeout_hours=1,\n",
+    "    label_column_name=label_column_name,\n",
+    "    track_child_runs=False,\n",
+    "    forecasting_parameters=forecasting_parameters,\n",
+    "    pipeline_fetch_max_batch_size=15,\n",
+    "    model_explainability=model_explainability,\n",
+    "    n_cross_validations=\"auto\",  # Feel free to set to a small integer (>=2) if runtime is an issue.\n",
+    "    cv_step_size=\"auto\",\n",
     "    # The following settings are specific to this sample and should be adjusted according to your own needs.\n",
-    "    \"iteration_timeout_minutes\": 10,\n",
-    "    \"iterations\": 10,\n",
-    "}\n",
+    "    iteration_timeout_minutes=10,\n",
+    "    iterations=10,\n",
```

Comment:
Ideally, we would like to set the number of iterations to 25. 10 seems too low given the first 7 are TS models and the last one is the voting ensemble. This leavers only 2 models recommended by MIRO.

Reply:
I have set it to 15 to avoid delays in daily builds.